server.py

```diff
@@ -28,6 +28,9 @@
 SERVER_LIST_FILE = 'servers.txt'
 LOG_DIRECTORY = 'logs'
 SERVER_CONFIG_FILE = 'server.cnf'
+DISTRIBUTED_GREEDY = 'greedy'
+DISTRIBUTED_REPLICATE = 'heuristic'
```

Comment:
this might be long and clunky, but I keep getting this confused with the method that's sent into the logs. Could we change this variable name into something else perhaps?